earth_enterprise/src/fusion/portableglobe/servers/fileunpacker/util.py

```diff
@@ -19,7 +19,12 @@
 
 import os
 import pexpect
-import pexpect.fdpexpect
+
+try:
```

Comment:
We should add a comment on why we're doing an import in a `try` statement. Something like, `# Try loading pexpect 3.`, `# Fall back to pexpect 2.`.